tests/unit/amo/api/test_users.js

```diff
@@ -134,4 +136,32 @@ describe(__filename, () => {
       mockApi.verify();
     });
   });
+
+  describe('userNotifications', () => {
+    const mockResponse = () => createApiResponse({
+      jsonData: createUserNotificationsResponse(),
+    });
+
+    const getParams = (params = {}) => {
```

Comment:
We've got the same `getParams` issue that I mentioned in another review, and I don't recall now what your response was, but my concern was the fact that we add all this code to have a flexible way of generating params for tests, but then never use any of it and just call `getParams` once.